Assets/Scripts/CountdownTimer.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using System;
+//not finished, need a visualization for the timer (ex. text)
```

Comment:
This code seems to interact with a visual component, so this comment should be removed